airbyte-cdk/python/reference_docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Why do we need this bat file? We already have makefile

Reply:
Both makefile and make.bat are only needed for the local development. So I left that if someone is going to use windows. I'm not sure if that's even the case, but that was my purpose to keep that file.